_posts/2021-03-16-notes-from-community-call-4-march-2021.md

```diff
@@ -0,0 +1,72 @@
+* [Elena Findley-de Regt](https://publiccode.net/team/elena-findley-de-regt.html)
+* [Boris van Hoytema](https://publiccode.net/team/boris-van-hoytema.html)
+* [Laura Scheske](https://publiccode.net/who-we-are/team/laura-scheske.html)
+* [Claus Mullie](https://publiccode.net/who-we-are/team/claus-mullie.html)
+* [Alba Roza](https://publiccode.net/who-we-are/team/alba-roza.html)
+* Johan Groenen
+* Martin Dias d’Ullois
+* Rasmus Frey
+
+## Updates from the Foundation
+
+* Version 0.2.1 of the Standard [is on its way](https://github.com/publiccodenet/standard/pull/478)
+* [Lightning talk video about the Standard ](https://www.youtube.com/watch?v=SKl3BX9kAbY) from FOSS Backstage is available
+* New episode of [Let's talk about public code](https://podcast.publiccode.net/e/4-lea-hemetsberger-open-agile-smart-cities/)
+
+## Proposed for this call
+
+We've had requests to review codebases against the Standard from codebases that at the time were not on a path towards full stewardship. We need to prioritize our work, but we also want the Standard for Public Code to be widely used.
+
+We've already created [an introduction course](https://citybycity.academy/course/standards-for-smart-cities) and briefly floated an idea of vendor training.
+
+Let's have an open discussion now on how to grow a community that's confident in applying the Standard for Public Code to a codebase.
+
+## Notes
+
+The discussion had a lot of different angles and ideas. Below are a few of those that sparked most discussion.
+
+### The standard as a help in procurement
+
+We discussed if the standard could be a help for a public organization about to procure development services. Right now the confidence of both buyer and the seller are too low for either of them to go that way. In theory, the standard should be able to be part of a procurement, much like a set of functional requirements. What might help are more codebases showing how they are compliant, and guides to get there.
+
+### "Flavors" of the standard
+
+Another idea that was discussed was "flavors" of the standard that certain solutions and practices are perscribed in a stronger sense than the standard is today. The openess of it now gives so many choices for the implementing party it doesn't really give much direction at all. If there were some framework specific flavors that perscribed certain styles, tools and methods, it would be easier to start developing and spend less time choosing amongst these.
+
+### Generic codebase template
+
+Related to this, but also different in the approach, one could imagine a very generic codebase (which in itself doesn't do much) that meets the standard that could be used as a reference point and inspiration. This could have the most preferred of all methods and good practices in it. But it might not be the Foundation for Public Code's role to maintain it or decide what goes in it, but rather it could be a community curated resource.
```

Comment:
```suggestion
Related to this, but also different in the approach, one could imagine a very generic codebase (which in itself doesn't do much) that meets the standard that could be used as a reference point and inspiration. This could have the most preferred of all methods and good practices in it. But it might not be the Foundation for Public Code's role to maintain it or decide what goes in it. Instead, it could be a community curated resource.
```